src/components/Article.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropsTypes from 'prop-types';
+
+export const Article = props => (
```

Comment:
maybe  destructuring assignment would be better?

Reply:
okey!